app/default/props.conf

```diff
@@ -10,12 +10,46 @@ TIME_FORMAT = %Y-%m-%dT%T.%6N%:z
 # Strip the RFC5424 header from the _raw field.
 SEDCMD-rfc5424_header = s/^\d+\s<\d{1,3}>\d\s(.+)\shost\s/\1 /
 
-TRANSFORMS-null = heroku_requests_doubleunderscore_null
+TRANSFORMS-source_type = heroku_system_source_type, heroku_app_source_type, heroku_requests_source_type, heroku_api_source_type, heroku_app_json_source_type
+
+[heroku_system]
+category = Structured
+description = Messages about actions taken by the Heroku platform infrastructure on behalf of your Heroku app.
+
+EXTRACT-heroku_log_format = ^\S+\s(?<component>\S+)\s(?<dyno>\S+)
+
+[heroku_requests]
+category = Structured
+description = Heroku app request logs produced by the Heroku HTTP routers.
+
+EXTRACT-heroku_log_format = ^\S+\s(?<component>\S+)\s(?<dyno>\S+)
+
+# Look up the Heroku error code description, see https://devcenter.heroku.com/articles/error-codes.
+LOOKUP-heroku_code_desc = heroku_error_lookup code OUTPUTNEW desc AS code_desc
+
+TRANSFORMS-heroku_requests_null_queue = heroku_requests_null_queue
+
+[heroku_api]
+category = Structured
+description = Messages about administrative actions taken by you and other developers working on your Heroku app.
+
+EXTRACT-heroku_log_format = ^\S+\s(?<component>\S+)\s(?<dyno>\S+)
+
+[heroku_app]
+category = Structured
+description = Logging output from your Heroku app, including logs generated by your code and dependencies.
+
+EXTRACT-heroku_log_format = ^\S+\s(?<component>\S+)\s(?<dyno>\S+)
+
+[heroku_app_json]
+category = Structured
+description = JSON based logging output from your Heroku app, including logs generated by your code and dependencies.
 
 EXTRACT-heroku_log_format = ^\S+\s(?<component>\S+)\s(?<dyno>\S+)
 
 # For JSON strip header values to enable syntax highlighting.
 EVAL-_raw = replace(_raw, "^.+\s-\s(\{.+\})\s*$", "\1")
 
-# Look up the Heroku error code description, see https://devcenter.heroku.com/articles/error-codes.
-LOOKUP-heroku_error = heroku_error_lookup code OUTPUTNEW desc AS code_desc
+# Based on the `_json` source type.
+INDEXED_EXTRACTIONS = json
+KV_MODE = none
```

Comment:
These don't seem to do anything.

I've also tried `KV_MODE = json`, no luck with that either.